templates/taxi/driver_detail.html

```diff
@@ -3,11 +3,18 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
   </h1>
-
+  
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
-  <p><strong>License number:</strong> {{ driver.license_number }}</p>
+  <p><strong>License number:</strong> {{ driver.license_number }}
+    <a style="margin-left: 20px" href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-primary">
```

Comment:
The URL name 'taxi:driver-update' might be misleading as it points to updating the driver's license. Consider renaming it to 'taxi:driver-license-update' for better clarity and consistency with the URL pattern.